salt/minion_auth/README.md

```diff
@@ -0,0 +1,47 @@
+# Foreman Salt Minion Authentication
+
+Currently, there are two possibilites to authenticate a newly deployed Minion automatically:
+1. Use the /etc/salt/autosign.conf file which stores the hostnames of acceptable hosts.
+2. Use Salt Autosign Grains for a more secure way which relies on a shared secret key.
+
+This README, handles the second option and how to configure it
+
+## Setup
+Add the content of 'master.snippet' to '/etc/salt/master' which configures the Grains key file on the Master and a Reactor. The Grains file holds the acceptable keys and will be written by the Smart Proxy when a new Minion is deployed. The Reactor initiates an interaction with Foreman Salt if a new Minion was authenticated successfully.
```

Comment:
which configuers the salt grains key file on the salt master and a salt reactor.

Reply:
Re-written with small letters.